resources/views/licenses/view.blade.php

```diff
@@ -378,8 +378,53 @@
 
               </div> <!-- end row-striped -->
             </div>
-            </div>
-          </div> <!-- end tab-pane -->
+
+          <div class="col-md-3">
+
+            @can('update', $license)
+              <a href="{{ route('licenses.edit', $license->id) }}" class="btn btn-block btn-primary" style="margin-bottom: 10px;">{{ trans('admin/licenses/general.edit') }}</a>
+              <a href="{{ route('clone/license', $license->id) }}" class="btn btn-block btn-primary" style="margin-bottom: 25px;">{{ trans('admin/licenses/general.clone') }}</a>
+            @endcan
+
+            @can('checkout', $license)
+              @if ($license->availCount()->count() > 0)
+                <a href="{{ route('licenses.checkout', $license->id) }}" class="btn-block btn bg-maroon" style="margin-bottom: 10px;" data-toggle="tooltip" title="{{ trans('general.checkout') }}">
+                  Checkout
+                </a>
+                <a href="#" class="btn-block btn bg-maroon" style="margin-bottom: 10px;" style="margin-bottom: 10px;" data-toggle="modal" data-tooltip="true"  data-target="#checkoutFromAllModal">Checkout Remaining to All Users</a>
+              @else
+                  <a href="{{ route('licenses.checkout', $license->id) }}" class="btn btn-block bg-maroon disabled">
+                    Checkout
+                  </a>
+                  <a href="#" class="btn btn-block bg-maroon disabled" style="margin-bottom: 10px;">
+                    Checkout Remaining to All Users
+                  </a>
+              @endif
+            @endcan
+
+            @can('checkin', $license)
+              @if (($license->seats - $license->availCount()->count()) > 0 )
+                <a href="#" class="btn btn-block bg-purple" style="margin-bottom: 10px;" data-toggle="modal" data-tooltip="true"  data-target="#checkinFromAllModal" data-toggle="tooltip" title="Checkin all checked out seats for this license">
+                  Checkin All
+                </a>
+              @else
+                  <a href="#" class="btn btn-block bg-purple disabled" style="margin-bottom: 10px;" data-tooltip="true" data-toggle="tooltip" title="This is disabled because there is nothing to checkin">
+                    Checkin All
+                  </a>
+              @endif
+            @endcan
+
+            @can('delete', $license)
+
+              @if ($license->availCount()->count() == 0)
```

Comment:
Should this be the same check as above, `$license->availCount()->count() == $license->seats`? Meaning, all seats are available?